packages/benchmark/README.md

```diff
@@ -0,0 +1,33 @@
+# n8n benchmarking tool
+
+Tool for executing benchmarks against an n8n instance.
+
+## Requirements
+
+- [k6](https://grafana.com/docs/k6/latest/)
```

Comment:
You mean docker image of `k6` or docker image of the benchmark cli?

Reply:
Of k6

Reply:
The benchmark cli is responsible for executing `k6`. Since we build a docker image of the benchmark cli, it's not really feasible to run a container from within another container.